src/o365/spo/commands/site/site-list-classic.ts

```diff
@@ -0,0 +1,223 @@
+            authorization: `Bearer ${auth.service.accessToken}`,
+            'X-RequestDigest': res.FormDigestValue
+          }),
+          body: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="2" ObjectPathId="1" /><ObjectPath Id="4" ObjectPathId="3" /><Query Id="5" ObjectPathId="3"><Query SelectAllProperties="true"><Properties /></Query><ChildItemQuery SelectAllProperties="true"><Properties /></ChildItemQuery></Query></Actions><ObjectPaths><Constructor Id="1" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}" /><Method Id="3" ParentId="1" Name="GetSitePropertiesFromSharePointByFilters"><Parameters><Parameter TypeId="{b92aeee2-c92c-4b67-abcc-024e471bc140}"><Property Name="Filter" Type="String">${Utils.escapeXml(args.options.filter || '')}</Property><Property Name="IncludeDetail" Type="Boolean">false</Property><Property Name="IncludePersonalSite" Type="Enum">${personalSite}</Property><Property Name="StartIndex" Type="String">${startIndex}</Property><Property Name="Template" Type="String">${webTemplate}</Property></Parameter></Parameters></Method></ObjectPaths></Request>`
+        };
+
+        if (this.debug) {
+          cmd.log('Executing web request...');
+          cmd.log(JSON.stringify(requestOptions));
+          cmd.log('');
+        }
+
+        return request.post(requestOptions);
+      })
+      .then((res: string): void => {
+        if (this.debug) {
+          cmd.log('Response:');
+          cmd.log(res);
+          cmd.log('');
+        }
+
+        const json: ClientSvcResponse = JSON.parse(res);
+        const response: ClientSvcResponseContents = json[0];
+        if (response.ErrorInfo) {
+          cmd.log(new CommandError(response.ErrorInfo.ErrorMessage));
+        }
+        else {
+          const sites: SPOSitePropertiesEnumerable = json[json.length - 1];
+          cmd.log(sites._Child_Items_.map(s => {
+            return {
+              Title: s.Title,
+              Url: s.Url
+            };
+          }).sort((a, b) => {
+            const urlA = a.Url.toUpperCase();
+            const urlB = b.Url.toUpperCase();
+            if (urlA < urlB) {
+              return -1;
+            }
+            if (urlA > urlB) {
+              return 1;
+            }
+
+            return 0;
+          }));
+        }
+        cb();
+      }, (err: any): void => this.handleRejectedPromise(err, cmd, cb));
+  }
+
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '-t, --type [type]',
+        description: 'type of classic sites to list. Allowed values are STS#0|BLOG#0|BDR#0|DEV#0|OFFILE#1|EHS#1|BICenterSite#0|SRCHCEN#0|BLANKINTERNET#0|BLANKINTERNETCONTAINER#0|ENTERWIKI#0|PROJECTSITE#0|PRODUCTCATALOG#0|COMMUNITY#0|COMMUNITYPORTAL#0|SRCHCENTERLITE#0|visprus#0|GROUP#0|SITEPAGEPUBLISHING#0.',
```

Comment:
Let's not list all web templates as they might change over time requiring us to always adjust our documentation